src/LSST/M1M3/SS/FPGA/SimulatedFPGA.cpp

```diff
@@ -49,6 +52,36 @@ SimulatedFPGA::SimulatedFPGA() {
         this->rnd[i] = float((rand() % 2000) - 1000) / 1000.0;
     }
     this->rndIndex = 0;
+    this->mgr_MTMount = SAL_MTMount();
+    this->mgr_MTMount.salTelemetrySub(const_cast<char*>("MTMount_Elevation"));
+
+    this->elevationReadWriteLock.unlock();
+
+    this->monitorMountElevationThread = std::thread(&SimulatedFPGA::monitorElevation, this);
+}
+
+SimulatedFPGA::~SimulatedFPGA() {
+    this->exitThread = true;
+
+    this->monitorMountElevationThread.join();
+}
+
+void SimulatedFPGA::monitorElevation(void) {
+    MTMount_ElevationC mountElevationInstance;
+
+    spdlog::debug("Start monitoring mount elevation...");
+
+    while (!this->exitThread) {
+        ReturnCode_t status = mgr_MTMount.getSample_Elevation(&mountElevationInstance);
+
+        if (status == 0) {
+            spdlog::debug("Got valid elevation sample...");
+            this->elevationReadWriteLock.lock();
+            this->mountElevation = mountElevationInstance.Elevation_Angle_Actual;
+            this->elevationReadWriteLock.unlock();
+        }
+        usleep(20000);
```

Comment:
usleep doesn't work with std::thread run (or some advanced thread options I forgot..but I remember I spend significant time debugging that with LBTO multithreads). Please replace with std::this_thread::sleep_for(20ms) - https://en.cppreference.com/w/cpp/thread/.